docs/setup-notes.md

```diff
@@ -57,23 +57,23 @@ After the teams have responded to the bugs reported by their testers, the issues
 This repository will then be used by the application for individual tester to respond to the team's assessment of the bug they had reported initially.
 
 ### Evaluation Repository
-After the testers have responded to the bug report responses. The issues and their respective information from the Tester's Response Phase will be transferred over to this repository. 
+After the testers have responded to the bug report responses. The issues and their respective information from the Tester's Response Phase will be transferred over to this repository.
 
 The application will then use this repository to post tutor's or admin's evaluation of each team's response and their respective bug report.
 
 ## Set up access rights
-For each of the teams, follow the steps below to assign the right access level to the repositories. 
+For each of the teams, follow the steps below to assign the right access level to the repositories.
 1. Go to their respective team page on Github.
 2. Click on `Repositories` navigation tab.
 3. Add the following repositories with the respective access right level as desribed below.
     1. For `admins` team.
-        1. pe: `Admin`
-        2. pe-results: `Admin`
-        3. pe-evaluation: `Admin`
+        1. Bug Reporting Phase / Tester's Response Phase Repository: `Admin`
```

Comment:
This is not necessary, right?

Reply:
Sorry prof, do you mean that this repository does not need admin?

Reply:
Yes, tester's repo is created by CATcher and Admin doesn't need any special access to it.

Reply:
Thanks for the clarification Prof! Will make the change.